appgate/client.py

```diff
@@ -117,6 +123,9 @@ async def get(self) -> Optional[List[Entity_T]]:
             return entities + self.magic_entities
         return entities
 
+    async def create(self, entity: Entity_T) -> None:
+        await self.post(entity)
```

Comment:
Is that aliases to match the k8s naming?
Because we use the client in both contexts right? SDP and k8s.

Reply:
yes, actually in 3 contexts now, SDP, k8s and git. Now the client interface is generic